packages/x-charts/src/LineChart/index.tsx

```diff
@@ -0,0 +1,2 @@
+export { default as LinePlot } from './LinePlot';
```

Comment:
I think we should stick with the format of the rest of X which is to avoid `export default`